src/views/instancetypes/list/InstanceTypePage.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { VirtualMachineInstancetypeModelRef } from '@kubevirt-ui/kubevirt-api/console';
+import VirtualMachineClusterInstancetypeModel, {
+  VirtualMachineClusterInstancetypeModelRef,
+} from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineClusterInstancetypeModel';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  ListPageCreate,
+  ListPageHeader,
+  useActiveNamespace,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Tab, Tabs, TabTitleText } from '@patternfly/react-core';
+
+import ClusterInstancetypeList from './ClusterInstancetypeList';
+import UserInstancetypeList from './UserInstancetypeList';
+
+import '@kubevirt-utils/styles/list-managment-group.scss';
+
+const InstanceTypePage = () => {
+  const { t } = useKubevirtTranslation();
+  const history = useHistory();
+  const [activeNamespace] = useActiveNamespace();
+  const [activeTabKey, setActiveTabKey] = useState<number | string>(
+    history?.location?.pathname.includes(VirtualMachineClusterInstancetypeModel.kind) ? 0 : 1,
+  );
+
+  const urlUserPreference = useMemo(
+    () =>
+      activeNamespace === ALL_NAMESPACES_SESSION_KEY
+        ? `/k8s/all-namespaces/${VirtualMachineInstancetypeModelRef}`
+        : `/k8s/ns/${activeNamespace}/${VirtualMachineInstancetypeModelRef}`,
+    [activeNamespace],
+  );
+
+  const groupVersionKind = useMemo(
+    () =>
+      activeTabKey === 0
+        ? VirtualMachineClusterInstancetypeModelRef
+        : VirtualMachineInstancetypeModelRef,
+    [activeTabKey],
+  );
+
+  useEffect(() => {
+    history.push(
+      activeTabKey === 0
+        ? `/k8s/cluster/${VirtualMachineClusterInstancetypeModelRef}`
+        : urlUserPreference,
+    );
+  }, [activeTabKey, history, urlUserPreference]);
+
+  return (
+    <>
+      <div className={classNames({ 'list-header-spacer': activeTabKey === 0 })}>
+        <ListPageHeader
+          title={
+            activeTabKey === 0
+              ? t('VirtualMachineClusterInstancetype')
```

Comment:
Why singular form used if it is a list of items we display?

Reply:
sure changed